war1tool.cpp

```diff
@@ -1926,8 +1926,8 @@ void MuxAllIntroVideos() {
 	const char* cmd4t = "ffmpeg -i %s/%s/title.avi -codec:v huffyuv "
 	    "%s/%s/title_s.avi"; // video filter crop not supported on windows
```

Comment:
```suggestion
	const char* cmd4t = "ffmpeg -i \"%s/%s/title.avi\" -codec:v huffyuv "
	    "\"%s/%s/title_s.avi\""; // video filter crop not supported on windows
```
Would also fix on windows for path with space.